pyproject.toml

```diff
@@ -33,7 +33,6 @@ classifiers = [
 dynamic = ["version"]
 dependencies = [
   'duckdb;python_version<"3.12"',
-  'duckdb==0.9.3.dev3597;python_version=="3.12"',
```

Comment:
Using `git blame` is usually a good proxy to understand the origin of changes, in this case it leads to the following: https://github.com/ImagingDataCommons/idc-index/commit/ac9705cb99de8d72bd96e372f8db464062551160

```
 fix: Support Python 3.12 adding dependency to duckdb pre-release

Since the latest regular release of duckdb does not project Python 3.12
wheels, we install the latest pre-release when Python 3.12 is used.
```

Reply:
Considering that duckdb 0.10.0 was released on Feb 13th, installing the pre-release is not needed anymore:

| | |
|--|--|
| ![image](https://github.com/ImagingDataCommons/idc-index/assets/219043/38198614-e1d0-433c-ab14-0085c49a5959) | ![image](https://github.com/ImagingDataCommons/idc-index/assets/219043/04726e74-e205-4b47-ba41-50e8b71219a1) |

Reply:
Nice! Thank you for the explanation JC!